src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -592,12 +592,15 @@ std::string ov::npuw::CompiledModel::funcall_mem_device(const std::size_t idx) c
 
 void ov::npuw::CompiledModel::remove_long_output_names(const std::shared_ptr<ov::Model>& model) {
     NPUW_ASSERT(model.get() != nullptr);
-    for (auto& output : model->outputs()) {
-        const auto& tensor_names = output.get_tensor().get_names();
-        if (tensor_names.size() > 32) {  // maximum supported
-            output.get_tensor().set_names({});
-            LOG_INFO("Removed output tensor names for " << model->get_friendly_name());
-            LOG_BLOCK();
+    for (auto node : model->get_ordered_ops()) {
+        for (size_t i = 0; i < node->outputs().size(); ++i) {
+            auto& output = node->output(i);
+            const auto& tensor_names = output.get_tensor().get_names();
+            if (tensor_names.size() > 32) {
+                output.get_tensor().set_names({});
+                LOG_INFO("Removed " << node->get_friendly_name() << " output tensor " << i << " names for " << model->get_friendly_name());
```

Comment:
It was already polluting the output with the `INFO` log level, I believe now it will print even more, let's move it to VERB.
Also, there's no need to mention the model name every time, you can do this just once before loop.
Also, the following `LOG_BLOCK()` makes no sense here